src/app/RootRoute.js

```diff
@@ -1,10 +1,10 @@
-// polyfill webpack require.ensure
-//if (typeof require.ensure !== 'function') require.ensure = (d, c) => c(require);
-
 import App from 'app/components/App';
 import PostsIndex from 'app/components/pages/PostsIndex';
 import resolveRoute from './ResolveRoute';
 
+// polyfill webpack require.ensure
+if (typeof require.ensure !== 'function') require.ensure = (d, c) => c(require);
```

Comment:
why are we doing this? condenser uses es6 modules. `require` shouldn't even be defined in the browser context.

Reply:
This part is being rendered both server and client side isomorphically.
In browser require.ensure is provided by webpack but when it renders on the server we just make it to use regular require.